docs/sources/operators-guide/configuring/reference-configuration-parameters/index.md

```diff
@@ -2717,6 +2717,12 @@ The `limits` block configures default and per-tenant limits imposed by component
 # CLI flag: -ingester.active-series-custom-trackers
 [active_series_custom_trackers_config: <map of tracker name (string) to matcher (string)> | default = ]
 
+# (experimental) Allow samples to be this old for out-of-order. Supported units:
+# h, m, s. If the value is non-zero, then the ingester will be enabled to query
+# overlapping blocks.
+# CLI flag: -ingester.ooo-allowance
+[ooo_allowance: <duration> | default = 0s]
```

Comment:
`outOfOrderAllowance` doesn't seem too verbose compared to some of the other limits. Could we use the longer form everywhere so that people don't need to translate "OOO" in their heads?

Reply:
Sure!